set.mm

```diff
@@ -19024,7 +19024,8 @@ Corresponds to the dual of Axiom (B) of modal logic.  (Contributed by NM,
     $( Version of ~ cbv3 with a disjoint variable condition, which does not
        require ~ ax-13 .  (Contributed by BJ, 31-May-2019.) $)
     cbv3v $p |- ( A. x ph -> A. y ps ) $=
-      ( wal nfal spimv1 alrimi ) ACHBDADCEIABCDFGJK $.
+      ( wal nf5ri hbal weq wi ax6ev eximii 19.36i alrimih ) ACHBDADCADEIJABCFCD
```

Comment:
The 3 steps ax6ev eximii 19.36i can still be abbreviated to spimv1

Reply:
You're right, I committed the suggestions in 18c5a03